test/acceptance/features/unbindAppToService.feature

```diff
@@ -106,3 +106,50 @@ Feature: Unbind an application from a service
 
         Then The env var "BACKEND_HOST" is not available to the application
         And The env var "BACKEND_USERNAME" is not available to the application
+    
+    Scenario: Unbind a generic test application from the backing service when the test application has been deleted
+        Given OLM Operator "backend" is running
+        * The Custom Resource is present
+            """
+            apiVersion: "stable.example.com/v1"
+            kind: Backend
+            metadata:
+                name: example-backend
+                annotations:
+                    service.binding/host: path={.spec.host}
+                    service.binding/username: path={.spec.username}
+            spec:
+                host: example.com
+                username: foo
+            """
+        * Generic test application "generic-app-a-d-u" is running
+        * Service Binding is applied
+            """
+            apiVersion: operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: binding-request-a-d-u
+            spec:
+                application:
+                    name: generic-app-a-d-u
+                    group: apps
+                    version: v1
+                    resource: deployments
+                services:
+                -   group: stable.example.com
+                    version: v1
+                    kind: Backend
+                    name: example-backend
+                    id: backend
+            """
+        * The application env var "BACKEND_HOST" has value "example.com"
+        * The application env var "BACKEND_USERNAME" has value "foo"
+
+        When App is deleted
```

Comment:
There is no need to provide the whole deployment yaml for deletion step.
We just need the name of the app

The step should be `App "generic-app-a-d-u" is deleted`

Reply:
@DhritiShikhar ,  I didn't find an existing step for `App "generic-app-a-d-u" is deleted` , then how to use it? 
  Did I miss anything?

Reply:
No @cdlliuy

Reply:
ok,  so since no established `App "generic-app-a-d-u" is deleted`,  and due to my limited knowledge on the acceptance test,  I  can't create a new step for this purpose...

I would like to continue to use yaml format for the `app deletion` step.

Reply:
> I would like to continue to use yaml format for the app deletion step.

@DhritiShikhar would you able to help perhaps?